open-craft/xblock-group-project-v2
# Displaying review status for individual teammates/groups in review stages

**Description:** This PR adds displaying review status for teammates/groups in teammate/group selector
**JIRA:** [MCKIN-3478](https://edx-wiki.atlassian.net/browse/MCKIN-3478)
**Testing instructions:**
0. Configure GPv2
1. Log in as student in a group with at least one other student; this student should not have completed peer grading and team evaluation reviews.
2. Open Team Evaluation stage (if there are multiple such stages - any would do just fine).
3. Note that Teammate selector now displays review state icons. 
4. Submit review; note that status icon have changed (also might make sense to refresh the page and make sure it is persisted)
5. Open Peer Grading stage.
6. Note that Group Selector now displays review state icons
7. Submit review; note that status icon have changed (also might make sense to refresh the page and make sure it is persisted)
